src/nvim/ex_docmd.c

```diff
@@ -5438,10 +5445,29 @@ static void ex_command(exarg_T *eap)
     return;
   } else {
     uc_add_command(name, end - name, p, argt, def, flags, compl, compl_arg,
-                   addr_type_arg, eap->forceit);
+                   addr_type_arg, eap->forceit, 0);
   }
 }
 
+void define_lua_command(const char *name,
```

Comment:
use the module common prefix.
```suggestion
void uc_define_lua_command(const char *name,
```